amplify/backend/api/elyze/schema.graphql

```diff
@@ -0,0 +1,83 @@
+type Proposition
+  @model
+  @auth(
+    rules: [
+      # allow IAM users to create, update, delete propositions
+      {
+        allow: private
+        provider: iam
+        operations: [create, update, delete, read]
+      }
+      # allow all guest users (not authenticated) to read posts
+      { allow: public, operations: [read] }
+    ]
+  ) {
+  id: ID!
+  idCandidat: Int
+  idTheme: Int
+  source: String
+  title: String
+  toBeShownFirst: Int
+  toShowOnSwipe: Int
+  firstPropositions: Int
+  createdAt: AWSDateTime
+  updatedAt: AWSDateTime
+  articleContent: String
+}
+
+type UserInfo
```

Comment:
I thought this data had been wiped out 🤔

Reply:
We actually have no idea if it has been wiped, since the actual schema was not made public.
Even though the app runs a query calling a mutation "deleteUserInfo", we have no way to tell if the corresponding mutation does anything (it might be empty) 🤡